src/Build/Evaluation/LazyItemEvaluator.RemoveOperation.cs

```diff
@@ -55,5 +72,23 @@ public ImmutableHashSet<string>.Builder GetRemovedGlobs()
                 return builder;
             }
         }
+
+        class RemoveOperationBuilder : OperationBuilder
+        {
+            public ImmutableList<string>.Builder MatchOnMetadata { get; } = ImmutableList.CreateBuilder<string>();
+
+            public MatchOnMetadataOptions MatchOnMetadataOptions { get; set; }
+
+            public RemoveOperationBuilder(ProjectItemElement itemElement, bool conditionResult) : base(itemElement, conditionResult)
+            {
+            }
+        }
+    }
+
+    public enum MatchOnMetadataOptions
```

Comment:
It would be nice to explicitly set a default when MatchOnMetadataOptions is not specified on the item. Right now it just happens to be CaseSensitive because that's how the  compiler assigns int values to the enums.

Reply:
Oops, no need, you already specified defaults :)